go/l1client/rollupcontractlib/README.md

```diff
@@ -0,0 +1,3 @@
+# Rollup contract library
+
+Handles transaction recovery and submission to and from the Rollup Contract.
```

Comment:
not sure what recovery means here. I guess you mean handles the processing of transactions to and from the "Managemet contract"
Because this will handle also the deposit transactions and the secret exchange transactions.
Basically all the management stuff.
The package should be renamed as well

Reply:
Recovered in the way  of having a blob of data and transforming it back to something meaningful. 
Let me rewrite that 👍 

I thought we were calling it rollup contract. Ense the name. Changing to management contract.

Reply:
this could use a better description. And it should talk about the management contract

Reply:
This issue is outdated, but I reviewed it and added a bit more of a description. I hope that description is enough for now.